<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChooseTime</class>
 <widget class="QWidget" name="ChooseTime">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>260</width>
    <height>109</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#label,#label_2,#start_date,#start_time,#end_date,#end_time,#ok,#cancel{
	font: 9pt &quot;微软雅黑&quot;;
}</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>238</width>
     <height>26</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="maximumSize">
       <size>
        <width>39</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>开始:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDateEdit" name="start_date"/>
    </item>
    <item>
     <widget class="QTimeEdit" name="start_time"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>238</width>
     <height>26</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="maximumSize">
       <size>
        <width>39</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>结束:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDateEdit" name="end_date"/>
    </item>
    <item>
     <widget class="QTimeEdit" name="end_time"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>70</y>
     <width>186</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QPushButton" name="ok">
      <property name="text">
       <string>确定</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="cancel">
      <property name="text">
       <string>取消</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
